notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-079_LUT_analysis_improvement.ipynb

```diff
@@ -34,7 +34,7 @@
     "warnings.filterwarnings('ignore')\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %matplotlib inline
</pre><p>I would remove this line and move the same line in the first cell, at the bottom of the first cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done.  Fix pushed.